src/Dictionaries/HashedDictionary.cpp

```diff
@@ -567,7 +567,7 @@ void HashedDictionary<dictionary_key_type, sparse>::calculateBytesAllocated()
 
             if constexpr (sparse || std::is_same_v<AttributeValueType, Field>)
             {
-                bytes_allocated += container.max_size() * (sizeof(KeyType) + sizeof(AttributeValueType));
+                bytes_allocated += container.size() * (sizeof(KeyType) + sizeof(AttributeValueType));
```

Comment:
I used max_size here, because it seems for sparse_hash_map, size is size of inserted elements, but max_size is like allocated capacity ?

Reply:
- max_size() - Returns the largest possible size of the sparse_hash_map
- bucket_count() - Returns table size, that includes empty and deleted
- size() - Returns table size, w/o empty and deleted

since this is sparsehash empty cells should not be significant and since items cannot be removed from the dictionary deleted is also not important.

Reply:
Okay. Thanks for explanation.